code/index.html

```diff
@@ -1,34 +1,57 @@
 <!DOCTYPE html>
 <html lang="en">
-
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="./style.css" />
     <link
       href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
-      rel="stylesheet" />
-    <title>Chat Bot</title>
+      rel="stylesheet"
+    />
+    <link
+    href="https://fonts.googleapis.com/css2?family=Lobster&display=swap"
+    rel="stylesheet"
+  />
+  <script
+  src="https://kit.fontawesome.com/7c916fe5d2.js"
+  crossorigin="anonymous"
+></script>
+
+    <title>Cafe Chat Bot</title>
   </head>
 
   <body>
-    <h1>Welcome to xxx.</h1>
-    <h2>Ask about xxxx.</h2>
-    <main>
-      <section class="chat" id="chat"></section>
-
+    <div>
+      <h1>Welcome to the Cafe</h1>
+      <h2>Let's find out what hot beverage you'd like to order today!</h2>
+      </div>
+    <div class="wrapper">
+       <main>
+      <section class="chat" id="chat"></section
       <div class="input-wrapper" id="input-wrapper">
-        <form id="name-form">
+        <button class="togo-btn" id="togo-btn">To Go</button>
+        <button class="sitin-btn" id="sitin-btn">Sit In</button>
+      </div>
+    
```

Comment:
I think you could have this buttons in the JavaScript as an innerHTML instead of the index.html.